clloader/scenarios/transformation_incremental.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable, List
+
+import numpy as np
+
+from clloader.task_set import TaskSet
+from clloader.datasets import _ContinuumDataset
+from clloader.scenarios import InstanceIncremental
+
+from torchvision import transforms
+
+class TransformationIncremental(InstanceIncremental):
```

Comment:
I'm not very aware of the litterature. I though that they were cases of rotation/permutation in NC mode?

Reply:
Yes, most of the time it is NC (I personally think that permutations and rotations are interesting in the NI scenario). I did not find a satisfying solution to have both easily but the goal is to have both

Reply:
I close this comment because I will open another PR with the renamed branch